src/akamai/js/akamaiHD_flash.js

```diff
@@ -583,6 +601,37 @@ require("../../../html5-common/js/utils/constants.js");
         width: event.eventObject.width,
         height: event.eventObject.height,
       }
+        var objectHeight = event.eventObject.height;
+        var objectWidth = event.eventObject.width;
+        var wrapperHeight = $(videoItem).parent().height();
+        var wrapperWidth = $(videoItem).parent().width();
+
+        $(videoItem).css("height", objectHeight+'px');
+        var resizeStyles = {};
+
+        if (wrapperHeight > (objectHeight*1.05)) {
```

Comment:
Added in:
        // The event.eventObject.[size] values seem to randomly fluctuate a lot,
        // so to prevent an accidental trigger of this formula from either that
        // or a rounding error we pad the smaller value up a little (1.05) to be sure the
        // larger value is large enough that the change is needed to ensure a
        // good viewing experience.